src/pseudopeople/configuration/validator.py

```diff
@@ -55,14 +55,20 @@ def _validate_noise_type_config(
     """
     for parameter, parameter_config in noise_type_config.items():
         parameter_config_validator = {
-            # todo add additional config value validators
-            Keys.POSSIBLE_AGE_DIFFERENCES: _validate_age_miswriting_perturbations_config
-        }.get(parameter, lambda *_: _)
+            (
+                NOISE_TYPES.age_miswriting.name,
+                Keys.POSSIBLE_AGE_DIFFERENCES,
+            ): _validate_possible_age_differences,
+            (
+                NOISE_TYPES.zipcode_miswriting.name,
+                Keys.ZIPCODE_DIGIT_PROBABILITIES,
+            ): _validate_zipcode_digit_probabilities,
```

Comment:
Are we not validating that simple omission probabilities aren't greater than 50%?

Reply:
No that went away: https://ihme.slack.com/archives/C02KUQ9LX32/p1681773592501719?thread_ts=1681765527.336939&cid=C02KUQ9LX32

Reply:
In that case, we don't need the key of this dictionary to be a tuple including the noise type. We can just go back to using the parameter alone as the key.

Reply:
ok, I figured I'd leave it as is for future-proofing but easy enough to change